#!/usr/bin/python3
def raise_exception_msg(message=""):
    try:
        print(a)
    except NameError:
        raise NameError(message)
